modules/op-release-notes-1-14.adoc

```diff
@@ -0,0 +1,290 @@
+
+* With this update, you can set the `builddefinition.buildtype` parameter in the `TektonConfig` CR to set the build type for in-toto attestation. When this parameter is `\https://tekton.dev/chains/v2/slsa`, {tekton-chains} records in-toto attestations in strict conformance with the SLSA v1.0 specification. When this parameter is `\https://tekton.dev/chains/v2/slsa-tekton`, {tekton-chains} records in-toto attestations with additional information such as the labels and annotations in each task run and pipeline run, and also adds each task in a pipeline run under `resolvedDependencies`.
+
+* Before this update, when {tekton-chains} was configured to use `gcs` storage, Tekton Chains did not record pipeline run information. With this update, {tekton-chains} records pipeline run information with this storage.
+
+* With this update, performance metrics are available for {tekton-chains}. To access the metrics, expose the `tekton-chains-metrics` service and then use the `/metrics` path on this service, port 9090. These metrics are also available in the {OCP} Monitoring stack.
+
+* With this update, {tekton-chains} uses the new `v2alpha3` record format version when recording pipeline runs and task runs that use the `v1` version value.
+
+* With this update, {tekton-chains} uses the `v1` version of pipeline run and task run formats internally.
+
+[id="tekton-results-new-features-1-14_{context}"]
+=== {tekton-results}
+
+* With this update, if {tekton-results} is installed, {tekton-results} records the summary and record data for pipeline runs started using {pac}.
+
+* With this update, {tekton-results} provides up to 100 megabytes of logging information for a pipeline or task.
+
+* With this update, any authenticated user can view the `tekton-results-api-service` route in the openshift-pipelines namespace to interact with Tekton Results using a REST API.
+
+* With this update, the {tekton-results} API includes a new endpoint for fetching summary and aggregation for a list of records.
+
+* With this update, the `GetLog` endpoint of the {tekton-results} API returns raw bytes with the `text/plain` content type.
+
+* With this update, you can optionally specify a custom CA certificate in the `options.configMaps.tekton-results-api-config.data.config.DB_SSLROOTCERT` spec in the `TektonResult` CR. In this case, Tekton Results requires an SSL connection to the database server and uses this certificate for the connection. However, if you use this spec, you must also use alternate configuration parameters for other {tekton-results} settings instead of the parameters described in the {tekton-results} documentation, as listed in the following table. Both the regular and the alternate parameter specs are within the `TektonResult` CR.
```

Comment:
can we leave out this part: `instead of the parameters described in the Tekton Results documentation`. Do we need a croos-ref here to understand the context better?
The third sentence sounds a bit confusing. 

Suggestion:
However, if you use this spec, you must also use alternate configuration parameters for other {tekton-results} settings as listed in the following table:

We can move the last line at the end of the table: Both the regular and the alternate parameter specs are defined within the `TektonResult` CR.

Also, use the attribute defined for Tekton results in this phrase:  `Tekton Results requires an SSL connection`